features/facebookImport/test/ministory/json-file-names-analysis.test.js

```diff
@@ -0,0 +1,86 @@
+import JSONFileNamesAnalysis from "../../src/model/analyses/report/json-file-names-analysis";
+import commonStructure from "../../src/static/commonStructure";
+import { ZipFileMock } from "@polypoly-eu/poly-import";
+import { runAnalysisForExport } from "../utils/analyses-execution";
+import { expectAnalysisSuccessStatus } from "../utils/analysis-assertions";
+
+describe("JSON files analysis for empty zip", () => {
+    let status = null;
+
+    beforeAll(async () => {
+        let zipFile = new ZipFileMock();
```

Comment:
Nit: For variables that don't get reassigned, we generally used `const` (i.e. wherever possible).